src/pages/landing/stocks/_parallelogram.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { graphql, useStaticQuery } from 'gatsby'
+import device from 'themes/device'
+import { Container, SectionContainer, Flex } from 'components/containers'
+import { Header, Text, QueryImage } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-grey-30);
+    clip-path: polygon(0 0, 100% 6%, 100% 100%, 0 94%);
+    padding: 6.4rem 1.6rem;
+    margin: 2rem auto;
+`
+const Content = styled(Flex)`
+    width: 60%;
+    flex-direction: column;
+    justify-content: center;
+    margin-right: ${(props) => props.margin_right};
+    margin-left: ${(props) => props.margin_left};
+
+    ${Text} {
+        margin-top: 2.2rem;
+        font-size: 2.4rem;
+        @media ${device.laptopM} {
+            font-size: 2.4rem;
```

Comment:
Is this required? As it was already declared as 2.4rem above.

Reply:
removed the font-size..